Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Add_IPs_to_Static_Address_Group.yml

```diff
@@ -0,0 +1,587 @@
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 340,
+          "y": -270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: eb27ca46-c816-4673-8290-e860fd2e2419
+    type: regular
+    task:
+      id: eb27ca46-c816-4673-8290-e860fd2e2419
+      version: -1
+      name: Get an Address Group
+      description: Lists all address groups.
+      script: '|||prisma-sase-address-group-list'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      folder:
+        complex:
+          root: inputs.Folder
+      name:
+        complex:
+          root: inputs.AddressGroupName
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 052be5c2-3071-431f-890a-e50ddc44cfe1
+    type: condition
+    task:
+      id: 052be5c2-3071-431f-890a-e50ddc44cfe1
+      version: -1
+      name: Was an IP provided?
+      description: Was an IP provided?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.IP
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 340,
+          "y": -140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: b0ac52d5-0098-41d0-8abf-429715147d43
+    type: title
+    task:
+      id: b0ac52d5-0098-41d0-8abf-429715147d43
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 340,
+          "y": 1480
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 57101bcc-571c-484a-8269-6ee1c31a9178
+    type: condition
+    task:
+      id: 57101bcc-571c-484a-8269-6ee1c31a9178
+      version: -1
+      name: Is there an Address Group?
```

Comment:
we can check if the address group and the IP together since with both the playbook will go to done

Reply:
Done